tools/scylla-nodetool.cc

```diff
@@ -2355,7 +2355,11 @@ class table_metrics {
     double compression_ratio() {
         return get<double>("compression_ratio");
     }
-    uint64_t estimated_row_count() {
+    uint64_t estimated_partition_count() {
+        // The API endpoint still uses "row" in "/metrics/estimated_row_count"
+        // although it returns partition counts. The endpoint name remains unchanged
```

Comment:
The name `row_count` is not incorrect. Historically, Cassandra use the term `row` for what we call a `partition` today. In time the terminology changed and some old code and associated comments can be really confusing in the way they use the term `row`.

Reply:
thank you Botond for sharing the context! now i have a better understanding of the context. so to be more accurate, the name is actually a little bit outdated instead of "incorrect".

Reply:
Yes. Still, it is good to update the term where possible, because whether you are aware of the historical context or not -- it is just confusing.